pyramid_closure/views.py

```diff
@@ -19,18 +17,21 @@ def depsjs(request):
     pyramid_closure = settings.get("pyramid_closure")
 
     roots = pyramid_closure.get("roots") if pyramid_closure else \
-        settings.get("pyramid_closure.roots")
-    roots = aslist(roots or [])
+        settings.get("pyramid_closure.roots", [])
+    roots = aslist(roots)
+
     for root in roots:
         path_to_source.update(depswriter._GetRelativePathToSourceDict(root))
 
     roots_with_prefix = pyramid_closure.get("roots_with_prefix") if \
         pyramid_closure else \
-        settings.get("pyramid_closure.roots_with_prefix")
-    roots_with_prefix = pairwise(roots_with_prefix or {})
-    for prefix, root in roots_with_prefix:
+        settings.get("pyramid_closure.roots_with_prefix", [])
+    roots_with_prefix = aslist(roots_with_prefix)
```

Comment:
I've also noticed that I still have issues to fix. I can't do it now, but I'll work on it this afternoon again.